thor/src/transaction.rs

```diff
@@ -436,16 +443,19 @@ impl SplitTransaction {
 
         let [(amount_0, address_0), (amount_1, address_1)] = outputs;
 
-        // Distribute transaction fee costs evenly between outputs
-        let half_fee = TX_FEE / 2;
+        // Distribute transaction TX_c fee costs evenly between outputs
+        let half_TX_c_fee = TX_c.fee() / 2;
+
+        // Distribute transaction TX_s fee costs evenly between outputs
+        let half_TX_s_fee = TX_s_fee / 2;
 
         let output_0 = TxOut {
-            value: amount_0.as_sat() - half_fee,
+            value: amount_0.as_sat() - half_TX_c_fee.as_sat() - half_TX_s_fee.as_sat(),
```

Comment:
That's a typo for sure.

Reply:
@luckysori

Reply:
If you're referring to `half_TX_c_fee`, it's not. If you pass in the amount that each party puts in the channel (in this case `amount_0`), you have to subtract both the party's fee contribution for `TX_c` and for `TX_s`.

Reply:
Ah well, this is the issue with having mono-letter variables (or close to).

Reply:
We can change that. I love long names.